deploy/LOCAL_INSTALL_HELM.md

```diff
@@ -0,0 +1,109 @@
+## Starting and Updating *seqr*
+
+Detailed instructions for how to install and update *seqr* may be found in the [seqr-helm](https://github.com/broadinstitute/seqr-helm) repository. 
+
+### Configuring Authentication for seqr
+
+#### Username/password basic auth
+This is the default authentication mechanism for seqr, and does not need any special steps for configuration.
+
+#### Google OAuth2
+Using Google OAuth2 for authentication requires setting up a Google Cloud project and configuring the seqr instance 
+with the project's client ID and secret by setting the following environment variables in your [helm values overrides](https://github.com/broadinstitute/seqr-helm?tab=readme-ov-file#valuesenvironment-overrides):
+```yaml
+  seqr:
+    environment:
+      - SOCIAL_AUTH_GOOGLE_OAUTH2_CLIENT_ID=your-client-id
+      - SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=your-client-secret
+```
+Note that user accounts do NOT need to be associated with this Google Cloud 
+project in order to have access to seqr. User's emails must explicitly be added to at least one seqr project for them to
+gain any access to seqr, and any valid Gmail account can be used.
+
+#### Azure OAuth2
+Using Azure OAuth2 for authentication requires setting up an Azure tenant and configuring the seqr instance with the 
+tenant and it's client ID and secret by setting the following environment variables in your [helm values overrides](https://github.com/broadinstitute/seqr-helm?tab=readme-ov-file#valuesenvironment-overrides):
+```yaml
+  seqr:
+    environment:
+      - SOCIAL_AUTH_AZUREAD_V2_OAUTH2_CLIENT_ID=your-client-id
+      - SOCIAL_AUTH_AZUREAD_V2_OAUTH2_SECRET=your-client-secret
+      - SOCIAL_AUTH_AZUREAD_V2_OAUTH2_TENANT=your-tenant-id 
+```
+Note that user accounts must be directly associated with the Azure tenant in order to access seqr. Anyone with access
+to the tenant will automatically have access to seqr, although they will only be able to view those projects that they 
+have been added to.
+
+## Enabling Clingen Allele Registration
+- Turning on this feature will register your variants within the Clingen Allele Registry during VCF ingestion. The [Registry](https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/landing) provides and maintains unique variant identifiers; enabling this feature will toggle several small features within *seqr*, including a link on the "Saved Variants" page to the Clingen Variant Curation Interface.
```

Comment:
The lunbk the VCI actually is shown whehter or not theres a CAID it just works differently. I think just saying "this feature will toggle several small features" is sufficient without giving any particular examples